build.sbt

```diff
@@ -1,22 +1,22 @@
-lazy val root = Project("root", file("."))
-  .dependsOn(MyBuild.netLogo)
+lazy val netLogoVersion = "6.1.1-RC1-664797c"
```

Comment:
So @LaCuneta, will I need to update this as we bump versions?

Reply:
Correct, that is what will control which NetLogo version since it is no longer just building off the `hexy` branch.